src/main/java/dev/su5ed/sinytra/connector/locator/ConnectorLocator.java

```diff
@@ -289,7 +290,9 @@ private static List<JarTransformer.TransformableJar> handleDuplicateMods(List<Ja
             .toList();
     }
 
-    private static boolean shouldIgnoreMod(String id, Collection<String> loadedModIds) {
+    private static boolean shouldIgnoreMod(ConnectorLoaderModMetadata metadata, Collection<String> loadedModIds) {
+        if (!metadata.loadsInEnvironment(FabricLoader.getInstance().getEnvironmentType())) return false;
```

Comment:
This should return `true` to stop a mod from loading in an incompatible environment

Reply:
Ok.